PanCardView/Common/CarouselView.cs

```diff
@@ -14,7 +14,9 @@ public CarouselView(IProcessor processor) : base(processor)
             // NOTE: For some reason setting this to True
             // Makes the view invisible on Android
             // Probably it's a MAUI bug
-            //IsClippedToBounds = true;
+            IsClippedToBounds = 
+                DeviceInfo.Platform == DevicePlatform.MacCatalyst || 
+                DeviceInfo.Platform == DevicePlatform.iOS;
```

Comment:
Would you like to set it to True and test it on Android?
Because this is supposed to be set to True
Check the Xamarin proj

https://github.com/AndreiMisiukevich/CardView/blob/master/PanCardView/CarouselView.cs#L14
Maybe the have fixed the bug

Reply:
Tested on all platforms using both repro project and sample project. Everything seems to be working fine with `IsClippedToBounds` set to `True`. Looks like they must have fixed whatever the issue was

Reply:
Thank you very much!